specification/billing/resource-manager/readme.go.md

```diff
@@ -1,46 +1,56 @@
-
```

Comment:
Is there a reason to make the Markdown file executable?

Reply:
Not sure what i did, but must have been system generated. The only change i did was add the path corresponding to the 2018-11-01-preview changes

Reply:
You can remove the bit by running `git update-index --chmod=-x FILENAME` to remove the executable bit and then commit it to undo the change

Reply:
Looks like it worked. The other Readmes touched also need the same fix